File: C/PRAK504-2210817210015-MuhammadIrfan.c
#include <stdio.h>
int reverse(int reverse){
//Lengkapi Function ini
    int n = 0;
    while(reverse!=0){
        n = n * 10;
        n = n + reverse % 10;
        reverse /= 10;}
    return n;}
int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));
}

File: Python/PRAK504-2210817210015-MuhammadIrfan.py
def reverse(reverse):
    r = 0
    while(reverse!=0):
        r = r * 10
        r = r + reverse % 10
        reverse //= 10
    return r
A, B = input().split()
A = reverse(int(A))
B = reverse(int(B))
C = A + B
print(reverse(C))